CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## [1.3.42](https://github.com/wordpress-mobile/AztecEditor-Android/releases/tag/v1.3.42)
+### Changed
+- Update Heading, Preformat, and HiddenHtml span classes to allow view level alignment (#899)
```

Comment:
Now that a lot more spans have received the two-classes treatment, maybe we need to mention them here?